packages/server/src/resources/resources.dto.ts

```diff
@@ -9,7 +9,7 @@ export class Resource {
     quantity: number;
     quantityCovered: number;
     price?: number | null;
-    state: ResourceStates;
+    state: ResourceStates | string;
```

Comment:
Strange doesn't look right that typescript not matching to enum

Reply:
I couldn't find an easy way to make make the compiler to match a string to the enum. Seems like it's not supported. Here is a discussion on that [topic](https://stackoverflow.com/questions/52370544/parse-string-as-typescript-enum)
We need it for testing only. I think we can

- Leave current changes
- Make the enum numeric as it was before (and make it less readable in debugging)
- Move the mock data from JSON file to an array in the code so that we can use enums